.gitignore

```diff
@@ -5,4 +5,8 @@ dataset_cache*
 runs/
 ParlAI/
 __pycache__
-.env/
\ No newline at end of file
+.env/
+runs/
+.DS_Store
+gpt_personachat_cache_v2/
```

Comment:
`gpt_personachat_cache_v2/` seems like it is something specific to your laptop, so you probably wanna drop it from the commit.